.github/workflows/silver_silicon_ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Run Backend CI
+
+on:
+  push:
+    branches: [ master ]
+  # `pull_request_target` can be triggered by a malicious PR to Silver
+  # So we run the backend CI on a fork to ensure safety
+  pull_request_target:
+    branches: [ master ]
+  workflow_dispatch:
+
+# `${{ secrets.PAT }}` must give access to `${{ env.fork }}/${{ matrix.repo }}`
+# for both pushes and triggering workflows (`workflow`+`repo` and `repo_hook` scopes).
+env:
+  fork: JonasAlaif
```

Comment:
I guess this should be adapted as well. We can use `viper-admin` for this too

Reply:
Yep, will do once those forks are created